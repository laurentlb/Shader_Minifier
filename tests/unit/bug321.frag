int n;
const float s = 1.;
float a(float s)
{
	s = 3.;
    n = 2.;
    return s;
}
float main()
{
	float t = 3.;
	t -= s;
    return a(s)+a(5.);
}
