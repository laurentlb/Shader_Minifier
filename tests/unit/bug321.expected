int n;
float a(float s)
{
  s=3.;
  n=2.;
  return s;
}
float main()
{
  float t=3.;
  t-=1.;
  return a(1.)+a(5.);
}